src/parser.js

```diff
@@ -471,12 +471,23 @@
     var nodelist = _getMultipleNodes(doc, ['use', 'svg:use']), i = 0;
     while (nodelist.length && i < nodelist.length) {
       var el = nodelist[i],
-          xlink = (el.getAttribute('xlink:href') || el.getAttribute('href')).substr(1),
+          xlinkAttribute = el.getAttribute('xlink:href') || el.getAttribute('href');
+
+      if (xlinkAttribute === null) {
+        return;
```

Comment:
what is this function returning on the others return early case? are we safe returning undefined?

Reply:
no explicit returns are there in the following code, so it's safe

Reply:
Only usage here: